mysql-test/suite/rocksdb/r/concurrent_alter.result

```diff
@@ -1,3 +1,6 @@
+set @orig_max_connections=@@global.max_connections;
+set @@global.max_connections=500;
+call mtr.add_suppression("Too many connections");
```

Comment:
I would leave the warning unsuppressed. 500 should be plenty, and if the warning is still seen, this would indicate something else needing investigating going on.